libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Views/SFSDKWindowManager.m

```diff
@@ -0,0 +1,304 @@
+/*
+ SFUIWindowManager.m
+ SalesforceSDKCore
+ 
+ Created by Raj Rao on 7/4/17.
+ 
+ Copyright (c) 2017-present, salesforce.com, inc. All rights reserved.
+ 
+ Redistribution and use of this software in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this list of conditions
+ and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright notice, this list of
+ conditions and the following disclaimer in the documentation and/or other materials provided
+ with the distribution.
+ * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
+ endorse or promote products derived from this software without specific prior written
+ permission of salesforce.com, inc.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+#import "SFSDKWindowManager.h"
+#import "SFSDKWindowContainer.h"
+#import "SFSDKWindow.h"
+#import "SFApplicationHelper.h"
+
+@interface SFSDKWindowManager() <SFSDKWindowContainerDelegate> {
+    SFSDKWindowContainer *_prevActiveWindow;
+    SFSDKWindowContainer *_currentWindow;
+}
+@property (nonatomic, strong) NSMutableOrderedSet *delegates;
+
+@end
+
+@implementation SFSDKWindowManager
+
+static const CGFloat SFWindowLevelAuth      = 300;
```

Comment:
> Windows with UIWindowLevelStatusBar and higher wont show the status bar.

👍 

Ah, looking at the constants usage, I realize these are just offsets.  Yeah, maybe rename them to indicate that the values are offset values, rather than assumed absolute window levels.